mkdocs/docs/experiments/categories/aws/ec2-stop-by-id.md

```diff
@@ -54,24 +54,24 @@ When the MANAGED_NODEGROUP is enable then the experiment will not try to start t
 
     ??? note "View the Minimal RBAC permissions"
 
-        [embedmd]:# (https://raw.githubusercontent.com/litmuschaos/chaos-charts/master/charts/kube-aws/ec2-terminate-by-id/rbac.yaml yaml)
+        [embedmd]:# (https://raw.githubusercontent.com/litmuschaos/chaos-charts/master/charts/kube-aws/ec2-stop-by-id/rbac.yaml yaml)
```

Comment:
we don't have rbac at this url. Please check at other places as well

Reply:
Removed all embedmd links for missing RBACs